server/src/main/resources/db/changelog/20140408160212-add-pool-source-sub-table.xml

```diff
@@ -87,7 +87,9 @@
     </changeSet>
 
     <changeSet id="20140408160212-3" author="ckozak">
-        <preConditions onSqlOutput="FAIL" onFail="CONTINUE"/>
+        <preConditions onSqlOutput="FAIL" onFail="CONTINUE">
+            <tableExists schemaName="candlepin" tableName="cp_pool"/>
```

Comment:
Is that schemaName correct/required? Don't think we use schema's anywhere do we?